src/khoj/routers/api.py

```diff
@@ -127,13 +127,15 @@ def _initialize_config():
             state.processor_config = configure_processor(state.config.processor)
 
     @api.get("/config/data", response_model=FullConfig)
+    @requires(["authenticated"], redirect="login_page")
```

Comment:
These API endpoints (apart from Web client endpoints) have been updated in next PRs, to not redirect to login page but return unauthorized http status code in response. So no change required here